tasks/sign-base64-blob/sign-base64-blob.yaml

```diff
@@ -75,7 +75,8 @@ spec:
         echo "done (${internalRequest})"
         
         payload=$(kubectl get internalrequest $internalRequest -o=jsonpath='{.status.results.signed_payload}')
+        decoded_payload=$(echo -n $payload | base64 -d)
```

Comment:
I found this in the bug report:

> The signature was not in binary format, it was base64 encoded.

I'm not sure if it's actually binary. But if it is, then we shouldn't save it as a string. Rather, we should save it to a file directly, e.g.:

```sh
payload=$(kubectl get internalrequest $internalRequest -o=jsonpath='{.status.results.signed_payload}')
checksum_file_name=$(ls $(workspaces.data.path)/$(params.binariesPath) | grep SHA256SUMS) 
base64 -d <<< "$payload" > $checksum_file_name
```

Reply:
I tested it in local with a payload and it can be saved in a bash variable. So it should work. But I don't mind doing it this way if preferable

Reply:
ok, in that case it's fine.